lib/movable_ink/version.rb

```diff
@@ -1,5 +1,5 @@
 module MovableInk
   class AWS
-    VERSION = '2.5.1'
+    VERSION = '2.5.2'
```

Comment:
```suggestion
    VERSION = '2.6.0'
```

We should also generate the lock file update

Reply:
👍  good call - looks good to me otherwise.